data.sql

```diff
@@ -1,35 +1,35 @@
 /* Populate database with sample data. */
- INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
  VALUES (1, 'Agumon', DATE '2020-02-03', 0, true, 10.23);
```

Comment:
- [ ] Well done removing the `ids` columns as requested by the previous reviewer 👏. However, kindly don't forget to do the same by removing the values that you gave to the IDs.  Otherwise, there will be a mismatch between the number of columns and the number of values you are trying to insert, which will likely result in an error. 💥
On this command, you should delete the value `1`. And do the same for the remaining commands at lines `6`,  `9`, `12`,  `17`,  `20`, `23`, `26`, `29`,  `32`, `35`.👌 😊

### Correct command :point_down: 

```
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Agumon', DATE '2020-02-03', 0, true, 10.23);
```